cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityUserOperator.java

```diff
@@ -167,6 +189,16 @@ public static EntityUserOperator fromCrd(Kafka kafkaAssembly) {
                 result.setLogging(userOperatorSpec.getLogging());
                 result.setGcLoggingEnabled(userOperatorSpec.getJvmOptions() == null ? true : userOperatorSpec.getJvmOptions().isGcLoggingEnabled());
                 result.setResources(userOperatorSpec.getResources());
+                if (kafkaAssembly.getSpec().getClientsCa() != null) {
+                    if (kafkaAssembly.getSpec().getClientsCa().getValidityDays() != null) {
+                        log.debug("Setting validity days to " + kafkaAssembly.getSpec().getClientsCa().getValidityDays());
```

Comment:
I don't think this really adds anything. None of the other places where we set a value in this method log anything.